charts/incubator/lldap/questions.yaml

```diff
@@ -4,41 +4,40 @@ portals:
 # Include{portalLink}
 questions:
 # Include{global}
-# Include{controllerExpert}
-# Include{fixedEnv}
-# Include{containerConfig}
-  - variable: env
-    group: "App Configuration"
-    label: "Image Environment"
-    schema:
-      additional_attrs: true
-      type: dict
-      attrs:
-        - variable: LLDAP_LDAP_USER_PASS
-          group: "App Configuration"
-          label: "Admin Password"
-          schema:
-            type: string
-            required: true
-            default: "change me"
-        - variable: LLDAP_LDAP_BASE_DN
-          group: "App Configuration"
-          label: "Base DN for LDAP"
-          schema:
-            type: string
-            required: true
-            default: "dc=example,dc=com"
+# Include{workload}
+# Include{workloadDeployment}
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+                                - variable: env
+                                  label: ""
+                                  group: "App Configuration"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                    - variable: LLDAP_LDAP_BASE_DN
+                                      label: "Base DN for LDAP"
+                                      schema:
+                                        type: string
+                                        required: true
+                                        default: "dc=example,dc=com"
+# Include{containerBasic}
```

Comment:
the format is off here by a huge margin

Reply:
If you look at Grafana the indent is pretty similar. Test SCALE Catalog fails with anything other than this indent. Unless it's better to move it to the container config include instead?

Reply:
it should be synced with grafana, yes.

Reply:
Thanks Ornias made that change now.